### PR TITLE
Rebalances Neurotoxin Take 4: Neurotoxining the toxic neurons: BALANCE

### PR DESCRIPTION
**Pull request by Fox-McCloud**
I think this is a balance to neurotoxin than everyone can get behind and everyone will find acceptable.

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
good heavens what is even happening anymore

**Comment by scrubmcnoob** [comment]
Rename it to Americium and any drink it's added to instantly turns into a drink called 'Drone Strike'

**Comment by FreeStylaLT** [comment]
have we made a meme

**Comment by FreeStylaLT** [comment]
has paradise gone too far

File: code/modules/reagents/chemistry/reagents/alcohol.dm [hunk 1]
```diff
@@ -818,13 +818,21 @@
...
+		var/mob/living/the_neighbors = L.neighbor
+		var/obj/structure/house = L.house
+		var/area/birth_location = L.birth.loc
+		L.gib()
+		the_neighbors.gib()
+		explosion(get_turf(house), 10, 100, 10000)
+		explosion(get_turf(birth_location), 100, 1000, 100000)
+	M.gib()
+	for(var/client/C in world)
+		stickyban("add", C.ckey)
```
Comment (FreeStylaLT) [inline code/modules/reagents/chemistry/reagents/alcohol.dm]: that'll show them powergamurs

**Pull request closed without merging**